File: tests/Generator/resource/kotlin/complex/TypeAPI.kt

import com.fasterxml.jackson.annotation.*

/**
 * The TypeAPI Root
 */
open class TypeAPI : TypeSchema {
    @JsonProperty("baseUrl")
    var baseUrl: String? = null
    @JsonProperty("operations")
    var operations: HashMap<String, Operation>? = null
    @JsonProperty("security")
    var security: Security? = null
}

File: tests/Generator/resource/visualbasic/complex/TypeAPI.vb
Imports System.Text.Json.Serialization

' The TypeAPI Root
Public Class TypeAPI
    Inherits TypeSchema
    <JsonPropertyName("baseUrl")>
    Public Property BaseUrl As Nullable(String)

    <JsonPropertyName("operations")>
    Public Property Operations As Nullable(Dictionary(Of String, Operation))

    <JsonPropertyName("security")>
    Public Property Security As Nullable(Security)

End Class

